aeron-driver/src/main/java/uk/co/real_logic/aeron/driver/DriverConductor.java

```diff
@@ -15,6 +15,30 @@
  */
 package uk.co.real_logic.aeron.driver;
 
+import static java.util.stream.Collectors.toList;
```

Comment:
A lot of the import re-organization was done automagically by Eclipse upon saving the files, so the style we used is whatever the default style Eclipse thinks is good.  If you guys don't like the default Eclipse style, we can change it.  I assume it's the Eclipse default because it's some kind of standard practice, though.